.gitignore

```diff
@@ -0,0 +1 @@
+code/secret.js
```

Comment:
Did you know you could add the .DS_Store file into your gitignore file as well so it doesn't get committed?